Sensorable/mobile/src/main/java/com/sensorable/services/EmpaticaTransmissionService.java

```diff
@@ -167,10 +168,9 @@ public void didReceiveBatteryLevel(float level, double timestamp) {
 
     @Override
     public void didReceiveTag(double timestamp) {
-        sendInfoMessage("RECEIVED TAG");
+        sendInfoMessage("Conexión estable");
     }
 
-
     @Override
     public void didEstablishConnection() {
         sendInfoMessage("CONNECTION ESTABLISHED");

```

Comment:
Let´s use the same language for all the messages. English or spanish but always the same.